python/lsst/ip/isr/isrTask.py

```diff
@@ -417,7 +425,7 @@ def readIsrData(self, dataRef, rawExposure):
             if self.config.doDark else None
         flatExposure = self.getIsrExposure(dataRef, self.config.flatDataProductName) \
             if self.config.doFlat else None
-        brighterFatterKernel = dataRef.get("brighterFatterKernel") if self.config.doBrighterFatter else None
+        brighterFatterKernel = dataRef.get("bfKernelNew") if self.config.doBrighterFatter else None
```

Comment:
I thought you decided to use "brighterFatterKernel"

Reply:
Indeed, this was just stale, thanks.